packages/dashboard/src/scss/main.scss

```diff
@@ -323,3 +324,7 @@
     margin: 1em 0;
   }
 }
+
+.margin-href {
```

Comment:
fix: avoid using !important
fix: use selector inside of the .cdc-open-viz-module namespace, if its within the dashboard nest inside that selector as well.

Reply:
OK yep moved that to _global and gave it a better name.  Technically it could be used for left margin on anything and not just an href... could also make it way longer name like download-link-margin-left if you prefer